src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1

```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.2.0'
```

Comment:
please make this version remain 0.1.0, Azure-PowerShell team will handle the version when release